internal/wasm/jit/jit_amd64.go

```diff
@@ -4702,7 +4700,7 @@ func (c *amd64Compiler) callFunction(addr wasm.FunctionAddress, addrReg int16, f
 	}
 
 	// 2) Set engine.valueStackContext.stackBasePointer for the next function.
-	{
+	if offset := (int64(c.locationStack.sp) - int64(len(functype.Params))); offset > 0 {
```

Comment:
if zero, the emitted code is virtually noop so I simply omit at the compilation phase